examples/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ set(subdirs
     throttle
     tuplespace
     transpose
+    fast_fourier_transformation
```

Comment:
Can you sort this alphabetically, please?

Reply:
Sure I'll do it